mmv1/products/compute/api.yaml

```diff
@@ -13012,18 +13012,16 @@ objects:
         properties:
           - !ruby/object:Api::Type::Integer
             name: 'vmCount'
-            at_least_one_of:
-              - group_placement_policy.0.vm_count
+            conflicts:
```

Comment:
From what I understand the `at_least_one_of` restriction actually allowed both `vm_count` and `availability_domain_count` to be set in the same config. Changing this to `conflicts` now introduces a new restriction that would make configs that did that invalid, thus a breaking change.

Could these fields absolutely not both be set? I would lean towards removing the restrictions entirely and letting the API handle the validation.

Reply:
These 2 fields can't be set (there is API validation for this). But you are right, it will be breaking change from TF perspective and right now it's better to remove all restrictions and rely on API validation.

Commit was updated.